proposals/NNNN-experiment-tracking/README.md

```diff
@@ -0,0 +1,495 @@
+# KEP-NNNN: Centralize Experiment Tracking in Kubeflow
+
+## Summary
+
+This proposal aims to resolve the current fragmented and limited experiment tracking experience by expanding the
+**Kubeflow Model Registry** into a unified, centralized metadata store. Currently, experiment tracking is scattered
+across components like Kubeflow Pipelines (which requires pipeline execution for tracking) and Katib (limited to
+hyperparameter tuning). This leads to challenges such as **limited flexibility** for direct logging from Python scripts
+or Jupyter notebooks, a **fragmented user experience** across multiple interfaces, and **maintenance difficulties** due
+to reliance on the inactive MLMD (Machine Learning Metadata) project.
+
+The proposal tackles these issues by:
+
+- **Expanding Model Registry** into a central experiment tracking store for experiments, runs, metrics, and artifacts
+  across all Kubeflow components.
+- Providing **MLFlow SDK compatibility**, enabling users to leverage familiar APIs while storing data in Kubeflow.
```

Comment:
Maybe rephrase this along the lines of "supporting third party" experimentation tracking. Mention MLFlow as a possible target instead of a goal?